docs/source/notes/extending.rst

```diff
@@ -121,6 +121,10 @@ non-Tensor arguments::
     track history. So if ``backward`` is implemented with differentiable
     operations, (e.g., invocation of another custom
     :class:`~torch.autograd.function`), higher order derivatives will work.
+    In this case, the Tensors saved with ``save_for_backward`` can also be used
+    in a differentiable manner but Tensor saved in the ``ctx`` cannot be used.
```

Comment:
"in a differentiable manner" - maybe clarify this? Can you be more specific about what tensors saved in 'ctx' cannot be used for? They cannot be part of a backward graph, right?